recipes/pytest-harvest/meta.yaml

```diff
@@ -0,0 +1,53 @@
+{% set name = "pytest-harvest" %}
+{% set version = "1.10.3" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: 532f99080aa85e8bfc7cb2ae572afaa77df727c4fc3a8f319f11669320cd4d24
+
+build:
+  noarch: python
+  number: 0
+  script: {{ PYTHON }} -m pip install . -vv
+
+requirements:
+  host:
+    - pip
+    - pytest-runner
```

Comment:
This shouldn't be needed to build the package.

Reply:
you are right. removed. thank you.